packages/react-native-renderer/src/ReactNativeAttributePayload.js

```diff
@@ -515,11 +452,8 @@ function addProperties(
   props: Object,
   validAttributes: AttributeConfiguration,
 ): null | Object {
-  if (enableAddPropertiesFastPath) {
-    return fastAddProperties(updatePayload, props, validAttributes);
-  } else {
-    return diffProperties(updatePayload, emptyObject, props, validAttributes);
-  }
+  // TODO: Fast path
```

Comment:
Remove this TODO if we're only going to do so in `ReactNativeAttributePayloadFabric`. I don't believe we should be making any other changes to Paper.

Reply:
This is just how the code looked before I added `fastAddProperties`.